riglist.go

```diff
@@ -10,7 +10,7 @@ package main
 import (
 	"fmt"
 	"strings"
-
+	"context"
```

Comment:
I believe we're missing a white space here. Did you [go fmt](https://go.dev/blog/gofmt) the code?

Reply:
added a commit after `go fmt`. It just reordered the entries